examples/login/login.py

```diff
@@ -1,61 +1,58 @@
-import time
+"""This example demonstrates how to log in to a website using AgentQL."""
 
-import agentql
+import time
 
-URL = "https://www.target.com"
-EMAIL = "REPLACE_WITH_YOUR_EMAIL (For target.com)"
-PASSWORD = "REPLACE_WITH_YOUR_PASSWORD (For target.com)"
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
 
-if __name__ == "__main__":
+URL = "https://www.yelp.com/"
+EMAIL = "REPLACE_WITH_YOUR_EMAIL (For yelp.com)"
+PASSWORD = "REPLACE_WITH_YOUR_PASSWORD (For yelp.com)"
 
-    # Start a session with the specified URL
-    session = agentql.start_session(URL)
-
-    # Define the queries to interact with the page
-    sign_in_query = """
-    {
-        sign_in_btn
-    }
-    """
+# Define the queries to interact with the page
+LOG_IN_QUERY = """
+{
+    log_in_btn
+}
+"""
 
-    credentials_query = """
-    {
+CREDENTIALS_QUERY = """
+{
+    sign_in_form {
         email_input
         password_input
-        sign_in_with_password_btn
+        log_in_btn
     }
-    """
+}
+"""
 
-    skip_query = """
-    {
-        skip_btn
-    }
-    """
-
-    # Make API call(s) to AgentQL server to fetch the query
-    response = session.query(sign_in_query)
-
-    response.sign_in_btn.click(force=True)
 
-    response_sign_in = session.query(sign_in_query)
+def main():
+    with sync_playwright() as playwright:
+        browser = playwright.chromium.launch(headless=False)
 
-    response_sign_in.sign_in_btn.click(force=True)
+        # Create a new page in the broswer and cast it to custom Page type to get access to the AgentQL's querying API
+        page: Page = browser.new_page()  # type: ignore
 
-    response_credentials = session.query(credentials_query)
+        page.goto(URL)
 
-    response_credentials.email_input.fill(EMAIL)
-    response_credentials.password_input.fill(PASSWORD)
-    response_credentials.sign_in_with_password_btn.click(force=True)
+        # Use query_elements() method to locate "Log In" button on the page
+        response = page.query_elements(LOG_IN_QUERY)
+        # Use Playwright's API to click located button
+        response.log_in_btn.click()
 
-    response_skip = session.query(skip_query)
+        # Use query_elements() method to locate email, password input fields, and "Log In" button in sign-in form
```

Comment:
@rachelnabors, need your help — I might be over optimizing here. I feel that email password input and "Log In" button should be named in one style (all capitalised or not, all in quotes or not, etc). How would you advise to name them in docs and examples?

Reply:
Ah I see! I think we do have other form filling examples -- one is `pricing_data.py` under ecommerce_pricing_data folder. Another one is `run_in_headless_browser.py` under run_script_in_headless_browser folder that is being reviewed in this PR:

https://github.com/tinyfish-io/fish-tank/pull/33

Reply:
If that's the case, I could start editing the example to demonstrate saving and load logged-in session.